README.md

```diff
@@ -123,6 +124,18 @@ CSV file seperated using commas. The first row should contain column names.
 ##### options:
 - path - path to file containing the data, required
 
+---
+#### Excel
+A single, coherent table in a Excel spreadsheet.
+The data may or may not have column names as first row.
+The data can be in any sheet, and start at any point, but it must
+be coherent (no spaces between rows, no spaces between columns).
```

Comment:
I think coherent is not the right word here. Gapless? Or maybe just write what you have in the parens.